From 535afceae976237b45da133254ef449b57ff96e4 Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Wed, 13 Oct 2021 10:17:53 +0200
Subject: [PATCH 1/3] Add support for libexif 0.6.14+ libexif 0.6.22+ was
 previously required because the macro EXIF_TAG_GAMMA was not defined reading
 the NEWS file.

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 31b4321..ee006be 100644
--- a/configure.ac
+++ b/configure.ac
@@ -141,7 +141,7 @@ fi
 # EXIF (optional)
 # ***************
 
-LIBEXIF_REQUIRED=0.6.22
+LIBEXIF_REQUIRED=0.6.14
 
 AC_ARG_WITH([libexif], AC_HELP_STRING([--without-libexif], [disable EXIF support]))
 have_exif=no
-- 
2.36.1

